xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -215,7 +320,7 @@
       section of the &sls; Release Notes:
```

Comment:
"consult the relevant section of the &sls; Release Notes" should be "consult the `<citetitle>ARM 64-Bit-Specific Features & Fixes (AArch64)</citetitle>` section of the &sls; Release Notes"

Reply:
I'll use the newly merged "Boot and Driver Enablement for Raspberry Pi" instead.